invenio_previewer/__init__.py

```diff
@@ -9,15 +9,18 @@
 r"""Invenio module for previewing files.
 
 Invenio-Previewer provides extensible file previewers for Invenio. It
-integrates with Invenio-Records-UI via a custom view function. Currently the
-module comes with viewers for the following files types:
+can be easily integrated with
+`Invenio-Records-UI <https://invenio-records-ui.rtfd.io>`_ to serve the file
+preview under an URL path like ``/records/<pid_value>/preview``.
+Currently the module comes with previewers for the following files types:
 
 - PDF (using PDF.js)
 - ZIP
 - CSV (using d3.js)
 - Markdown (using Mistune library)
 - XML and JSON (using Prism.js)
 - Simple images (PNG, JPG, GIF)
+- IPython Notebooks
```

Comment:
In the `Configuration` section (`L40`) when we mention "adding Records-UI endpoint" we could add a link to [`invenio_records_ui.config.RECORDS_UI_ENDPOINTS`](https://invenio-records-ui.readthedocs.io/en/latest/configuration.html#invenio_records_ui.config.RECORDS_UI_ENDPOINTS) so it is clear which API does it offer.